src/main/java/callhub/connect/entities/Message.java

```diff
@@ -2,25 +2,23 @@
 import org.springframework.data.annotation.Id;
 import org.springframework.data.mongodb.core.mapping.Document;
 
-import java.text.DateFormat;
-import java.text.SimpleDateFormat;
-import java.time.LocalDate;
-import java.util.Date;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
 
 @Document("message")
 public class Message {
 
     @Id
     private String id;
     private String content;
-    private LocalDate timeStamp;
+    private LocalTime timeStamp;
```

Comment:
I didn’t want to have the day/month/year in the transcript, because it was too repetitive, but I can see if I can use LocalDateTime, but still have the transcript only have the time.

Reply:
I would suggest putting the date at the top of the email, that way it's consistent